Makefile

```diff
@@ -1,11 +1,24 @@
 ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
+# All components are prefixed by st2
+COMPONENTS := $(wildcard /tmp/st2/st2*)
```

Comment:
NVM. Saw that you are cloning st2 to /tmp. 

So the reason you need to clone st2 is because pylint would complain about import errors otherwise?